pkg/osutil/interrupt_unix.go

```diff
@@ -42,10 +43,10 @@ func RegisterInterruptHandler(h InterruptHandler) {
 	interruptHandlers = append(interruptHandlers, h)
 }
 
-// HandleInterrupts calls the handler functions on receiving a SIGINT or SIGTERM.
+// HandleInterrupts calls the handler functions on receiving a SIGINT, SIGQUIT or SIGTERM.
```

Comment:
do not change this part. go should handle sigquit by default. it should just print out the stack trace.